CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(uvc_ros_driver)
 
-# if traget is an ARM processor uncomment this line to compile with NEON instruction
+# If traget is an ARM processor uncomment this line to compile with NEON instruction.
```

Comment:
I don't see NEON being enabled here - could you comment on the intent? Is there an easy way to scan for NEON support with CMAKE?

Reply:
also should be `target` and not `traget`. @LorenzMeier I guess the purpose was only to update the comments.

Reply:
For enabling NEON, it should be set `(NEON_FLAGS "-DENABLE_NEON -mfloat-abi=hard -mfpu=neon-vfpv4 -mcpu=cortex-a15 -Ofast")`, as bellow. What I think it could be done is check the for the target architecture automatically instead of manual set, like checking for `CMAKE_SYSTEM_PROCESSOR` or something. @fabianbl I guess a more complete upgrade could include that.

Reply:
@LorenzMeier @TSC21 Ok, I added a cmake scripts which checks if an ARM architecture supporting Neon instructions is available.